docs/content/guides/columns/column-filter.md

```diff
@@ -266,7 +266,7 @@ const hot = new Handsontable(container, {
 ::: only-for react
 ::: example #example3 :react
 ```jsx
-import React, { useEffect } from 'react';
+import React, { useEffect, useRef } from 'react';
```

Comment:
It should work probably.

```suggestion
import { useEffect, useRef } from 'react';
```

Reply:
that's true, looks like we do not need this import here.